webpack/assets/javascripts/react_app/components/notifications/notifications.scss

```diff
@@ -1,6 +1,6 @@
 #notifications_container {
   position: static;
-  margin-top: 2px;
+  margin-top: -1px;
```

Comment:
please make sure the notification drawer opens below the top bar, it currently opens in the middle of the bar
![image](https://user-images.githubusercontent.com/20167/30902328-84a07696-a373-11e7-897a-78a3c791e791.png)